packages/api/src/middlewares/withFilters.ts

```diff
@@ -155,7 +155,7 @@ export const withFilters = t.middleware(({ next, input = {} }) => {
       filters.transactionAddresses = [];
       if (from) {
         filters.transactionAddresses.push({
-          fromId: from,
+          fromId: { in: from.split(",") },
```

Comment:
We should check filter out empty strings when splitting by separator

Reply:
Empty strings `from` addresses have been removed 👍 

If we send `,,,,,` as `from` query param, the API should return all the data without this filter instead of an empty list